addons/api/fnc_isRadio.sqf

```diff
@@ -13,16 +14,16 @@
  *
  * Public: Yes
  */
-#include "script_component.hpp"
 
-params ["_weapon"];
+params [
+    ["_weapon", "", [""]]
+];
 
 // check that it at least inherits from ACRE_BaseRadio *AND* has a unique ID
 // If its scoped 1, that means its either a base or not a radio.
 private _flag = getNumber (configFile >> "CfgWeapons" >> _weapon >> "acre_isRadio");
 private _id = getNumber (configFile >> "CfgWeapons" >> _weapon >> "acre_uniqueId");
 private _scope = getNumber (configFile >> "CfgWeapons" >> _weapon >> "scope");
 if (isNil "_flag" || isNil "_scope" || isNil "_id") exitWith { false };
-if (_flag == 1 && _scope == 1) exitWith { true };
 
-false
+(_flag == 1) && (_scope == 1)
```

Comment:
Could do `_flag && _scope`, no? Unsure of readability though. :D

Reply:
I prefer like it is now just for readability.